packages/tpetra/core/test/FECrsGraph/FECrsGraph_UnitTests.cpp

```diff
@@ -338,11 +338,17 @@ TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL( FECrsGraph, Assemble1D_LocalIndex, LO, GO, No
 // INSTANTIATIONS
 //
 
+#ifdef HAVE_TPETRA_EXPERIMENTAL
```

Comment:
If experimental is disabled, you might get a lot of "unused classes" warnings in this .cpp file.

Reply:
Good catch!